src/test/java/healthy/lifestyle/backend/workout/service/ExerciseServiceTest.java

```diff
@@ -168,4 +169,32 @@ void getExercisesTest_shouldReturnCustomExercises() {
                     .isEqualTo(exercisesDtoActual.get(id).getHttpRefs());
         });
     }
+
+    @Test
+    void getExerciseTest_shouldReturnDefaultExercise() {
+        // Given
+        long exercise_id = 1L;
+        Exercise exercise = dataUtil.createExercise(exercise_id, false, false, false, 1, 2, 1, 2);
+        when(exerciseRepository.findDefaultById(exercise_id)).thenReturn(exercise);
+
+        // When
+        ExerciseResponseDto exercisesDtoActual = exerciseService.getDefaultExerciseById(exercise_id);
```

Comment:
Please replace `exercisesDtoActual` with `exerciseDtoActual` since a name has to point to the meaning.